saleor/graphql/core/enums.py

```diff
@@ -11,6 +11,35 @@ class ReportingPeriod(graphene.Enum):
     THIS_MONTH = 'THIS_MONTH'
 
 
+def to_enum(enum_cls, *, type_name=None, **options) -> graphene.Enum:
```

Comment:
Why do we need this function and what were the consequences of not having it before? Are there any problems with enums currently?

Reply:
It avoids to repetitively have
```python
graphene.Enum('something', 
    str_to_enum(code.upper()), code) for code, name in enum_cls.CHOICES)
```

It would make it more readable and easier to maintain.